extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -404,10 +404,16 @@ export class AmpA4A extends AMP.BaseElement {
       // onLayoutMeasure gets called multiple times.
       return;
     }
-    this.layoutMeasureExecuted_ = true;
+    const slotRect = this.getIntersectionElementLayoutBox();
+    if (slotRect.height == 0 || slotRect.width == 0) {
```

Comment:
Should there be an explicit check for `display:none` here?  Or is that subsumed by the dimension checks?